src/models/project.ts

```diff
@@ -1,15 +1,15 @@
-namespace App {
-  export enum ProjectStatus { //add two more to do and for review from review only to finished or active
-    Active,
-    Finished,
-  }
-  export class Project {
-    constructor(
-      public id: string,
-      public title: string,
-      public description: string,
-      public people: number,
-      public status: ProjectStatus
-    ) {}
-  }
-}
\ No newline at end of file
+export enum ProjectStatus {
+  Active,
```

Comment:
it is ok to have enum and not a custom type, if you are  leaving the enum i think you need to have string values for each state so you can map those easily in the code